### PR TITLE
Add Dig Chain

### PR DESCRIPTION
**Pull request by MasterPi-2124**
Hi, we are Notional and here is Dig Chain. Please review us!

### CODE REVIEW

File: dig/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "dig-1",
+    "assets": [
+      {
+        "description": "",
```
Comment (sunnya97) [inline dig/assetlist.json:6]:
```suggestion
        "description": "Native token of Dig Chain",
```

File: dig/chain.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  "chain_id": "dig-1",
+  "bech32_prefix": "dig",
+  "daemon_name": "digd",
+  "node_home": "$HOME/.digd",
+  "genesis": {
+    "genesis_url":  "https://raw.githubusercontent.com/notional-labs/dig/master/networks/mainnets/dig-1/genesis.json"
+  },
+  "key_algos": [
+    "secp256k1",
+    "eth-secp256k1"
```
Comment (sunnya97) [inline dig/chain.json:16]: This is supported?

File: dig/chain.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      {
+        "id": "eb55b70c9fd8fc0d5530d0662336377668aab3f9",
+        "address": "185.194.219.128:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://65.21.202.37:8001"
```
Comment (sunnya97) [inline dig/chain.json:84]: This only works over http, not https.

Can you add an https endpoint please?

File: dig/chain.json (hunk 2 above)
Comment (MasterPi-2124) [reply dig/chain.json:16]: Hi @sunnya97, eth-escp256k1 is supported. We are working to support clients using eth addresses

File: dig/chain.json (hunk 3 above)
Comment (MasterPi-2124) [reply dig/chain.json:84]: opps sorry, Let me change that. For now we just launched for a day and do not have a strong server yet, so we dont have many other servers. We will add them very soon when we have one.

**Comment by sunnya97** [comment]
Ok, I will merge as is for now.  Please make a PR when you have an https endpoint

**Pull request merged**